integration-tests/src/test/resources/features/action-webhook.feature

```diff
@@ -67,15 +67,15 @@ Feature: Webhook Action tests
         "action": {
           "type": "webhook_sink_0.1",
           "parameters": {
-              "endpoint": "https://webhook.site/${env.webhook.site.uuid.second}"
+              "endpoint": "https://webhook.site/${webhook.site.token.second}"
           }
         },
         "transformationTemplate" : "{\"text\": \"hello {data.name} by {id}\"}"
       }
       """
 
     And the Processor "testProcessor" of the Bridge "mybridge" has action of type "webhook_sink_0.1" and parameters:
-      | endpoint | https://webhook.site/${env.webhook.site.uuid.second} |
```

Comment:
I'm wondering about uuid now. Was there any random part of the webhook site previously? I'm not aware about any as both webhooks were specified in the test configuration. But I would like to ask to not to be sure.

Reply:
no, there was no random Webhook, all were predefined